test/jest/victory-candlestick/helper-methods.test.js

```diff
@@ -0,0 +1,77 @@
+/* eslint max-nested-callbacks: 0 */
+import { helperMethods } from "victory-candlestick";
+import { range } from "lodash";
+import { fromJS } from "immutable";
+
+const immutableGetDataTest = {
+  createData: (x) => fromJS(x),
+  testLabel: "with immutable data"
+};
+const getDataTest = {
+  createData: (x) => x,
+  testLabel: "with js data"
+};
+
+const { getData, getDomain } = helperMethods;
+
+[getDataTest, immutableGetDataTest].forEach(({ createData, testLabel }) => {
```

Comment:
I know this is how the original test file was set up, but since there are only a few tests here personally I would just copy/paste the test assertions and add an extra "describe" block for immutable data. What do you think @heythisispaul ?

Reply:
Yeah I'm good with that, I think that's a little more declarative of an approach.

Reply:
I ended up making this change, found it to be a lot of code duplication, made some more adjustments and pretty much just wound back up with something very similar to this implementation so I decided to just keep it. I'm definitely still open to shaking it up if you have any specific ideas though!

Reply:
No problem, we can keep it the same for now.